_translator-files/README.md

```diff
@@ -1,17 +1,14 @@
 # Website docs translation:
 
-All the files for translators to work on are contained in a single directory: `_translator-files/po/YOUR-LANGUAGE/`. **Please do not edit any other files**. When adding a new language, all the necessary files will be automatically created and modified.
+The recommended way to translate the website documentation is by using [Weblate](https://hosted.weblate.org/projects/jamulus/). If you are comfortable using GitHub and .po file editors, and still want to contribute your translation via a Pull Request, the files for each language are contained in: `_translator-files/po/LANGUAGE/`. You can find a guide on .po file editors [here](https://github.com/jamulussoftware/jamuluswebsite/tree/release/_posts2022-08-04-Translating-po-files.md). Please make your PR against the `next-release` branch.
```

Comment:
There's a slash missing. Also try to link to jamulus.io if it's part of the KB, not the raw .md files

Reply:
Done.